includes/query_processor.inc

```diff
@@ -236,6 +236,19 @@ class IslandoraSolrQueryProcessor {
       $params_array = array_merge($params_array, $params_date_facets);
     }
 
+    // Determine the default facet sort order.
+    // https://cwiki.apache.org/confluence/display/solr/Faceting#Faceting-Thefacet.sortParameter
```

Comment:
Line 240 is giving "Line exceeds 80 characters; contains 96 characters". Since It's a link, so not sure how you want to split that into 2 lines.